Vagrantfile

```diff
@@ -43,8 +43,10 @@ BROADCAST_ADDRESS = n | (~n.instance_variable_get(:@mask_addr) & IPAddr::IN4MASK
 CLUSTER_IP_CIDR = settings["pod_network"]["cluster_ip_cidr"]
 SERVICE_IP_CIDR = settings["pod_network"]["service_ip_cidr"]
 
+VAGRANT_PROVIDER = settings["conf"]["vagrant_provider"]
```

Comment:
You need to add the instructions in the README about installing vagrant-libvirt and its dependencies